src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,29 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        int[] salaryAll = new int[names.length];
+        for (int i = 0; i < names.length; i++) {
+            for (String datas : data) {
+                String[] datasPart = datas.split(" ");
+                DateTimeFormatter formatter = DateTimeFormatter.ofPattern("d.MM.yyyy");
+                LocalDate dateFromLocal = LocalDate.parse(dateFrom, formatter);
+                LocalDate dateToLocal = LocalDate.parse(dateTo, formatter);
```

Comment:
you don't need to calculate them on every loop step. move variables outside of the loop